Source/OrchBase/OrchBasePlace.cpp

```diff
@@ -0,0 +1,204 @@
+/* Defines the behaviour of "placement" commands from the operator. A synopsis
+ * can be found in the placement documentation (or talk to MLV). */
+
+/* Everything goes through here. always returns zero. */
```

Comment:
Somewhat - the caller (`Root::ProcCmnd`) checks the return value and closes on non-zero. Since the return value of `OrchBase::Cm*` commands is always treated the same, I thought consistency wins out.